src/Test/Utilities/Portable/CompilationVerifier.cs

```diff
@@ -151,6 +152,37 @@ void listMethodsInType(ICSharpCode.Decompiler.TypeSystem.ITypeDefinition type, D
             }
         }
 
+        public void VerifyTypeIL(string typeName, string expected)
+        {
+            var output = new ICSharpCode.Decompiler.PlainTextOutput();
+            using (var testEnvironment = RuntimeEnvironmentFactory.Create(_dependencies))
+            {
+                string mainModuleFullName = Emit(testEnvironment, manifestResources: null, EmitOptions.Default);
```

Comment:
Maybe the emit options should be passed in, so you can output release/debug?

Reply:
Isn't that a part of CompilationOptions? `OptimizationLevel`?